e2e/cypress/integration/api/iam/iam_integration.spec.ts

```diff
@@ -344,6 +377,42 @@ describeIfIAMV2p1('assigning projects', () => {
       });
     });
 
+    // (TC) We do allow users to remove their project, even if results in unassigned and
+    // they don't have iam:project:assign perms on unassigned (special case).
+    it('assigning authorized to (unassigned) is allowed ' +
```

Comment:
I think this tests the same: https://github.com/chef/automate/blob/f1a7c43f5874872b9b9876aaf8d5017961f40119/e2e/cypress/integration/api/iam/iam_integration.spec.ts#L463-L476